app/components/base/Card.tsx

```diff
@@ -148,4 +107,72 @@ const DescriptionStyle = styled.div`
   }
 `;
 
+type Props = {
+  link?: { href?: string; external?: boolean };
+  centerAlign?: boolean;
+  isFlex?: boolean;
+  icon: string;
+  badge?: string;
+  topic: string;
+  description: string;
+};
+
+const CoreCard = ({
+  description,
+  badge,
+  centerAlign,
+  isFlex,
+  icon,
+  link,
+  topic,
+}: Props) => (
+  <CardStyle className={classnames({ center: centerAlign }, { flex: isFlex })}>
+    <StyledIcon
+      src={icon}
+      alt=""
+      className={classnames({ center: centerAlign })}
+    />
+
+    {badge && (
+      <BadgeStyle>
+        <Badge text={badge} />
+      </BadgeStyle>
+    )}
+
+    {link && link.external && (
+      <LinkIconStyle>
+        <NewTabIcon width={13} />
+      </LinkIconStyle>
+    )}
+    <ConditionalWrapper
```

Comment:
Added a ConditionalWrapper Util component that wraps elements based on a condition. In this case, the `Topic` and `Description` are wrapped in a `<div>` when "isFlex" is present.